src/main/java/com/meilisearch/sdk/KeysHandler.java

```diff
@@ -0,0 +1,90 @@
+package com.meilisearch.sdk;
+
+import com.google.gson.*;
+import com.google.gson.reflect.TypeToken;
+import com.meilisearch.sdk.exceptions.MeiliSearchApiException;
+
+/**
+ * Wrapper around MeilisearchHttpRequest class to use for MeiliSearch keys
+ *
+ * <p>Refer https://docs.meilisearch.com/reference/api/keys.html
+ */
+public class KeysHandler {
+    private final MeiliSearchHttpRequest meilisearchHttpRequest;
+    private final Gson gson =
+            new GsonBuilder().serializeNulls().setDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'").create();
+
+    /**
+     * Creates and sets up an instance of Key to simplify MeiliSearch API calls to manage keys
+     *
+     * @param config MeiliSearch configuration
+     */
+    public KeysHandler(Config config) {
+        this.meilisearchHttpRequest = new MeiliSearchHttpRequest(config);
+    }
+
+    /**
+     * Retrieves the Key with the specified uid
+     *
+     * @param uid Identifier of the requested Key
+     * @return Key instance
+     * @throws Exception if client request causes an error
+     */
+    Key getKey(String uid) throws Exception, MeiliSearchApiException {
```

Comment:
In this java version we don’t need to define the visibility of the method? `public, private...`

Reply:
I think we should. I should add it to the `TaskHandler` too but there is none on the `IndexHandler` so I didn't add it.